### ISSUE TITLE
[Feature Request] Remove space before digits or replace with '-' when saving multiple frames as png.

### ISSUE THREAD

**Issue by agaertner** [NickeManarin/ScreenToGif#1116]
Just a very small thing. When saving an animation per frame into multiple pngs the enumeration is suffixed with a leading space.

It is prone to be accidentally removed when referencing in LaTeX or other software where extensions are omitted.

This is how one inserts 46 frames (``react-page-frame <00-45>.png``) as an animation into a PDF using LaTeX
```tex
\usepackage{animate}
\animategraphics[controls,loop,autoplay,poster=15,width=\linewidth]{12}{./img/react-page-anim/react-page-frame }{00}{45}
```
Notice the path ending with a space because of the file names containing a space (eg. ``react-page-frame 01.png``).
I would rather have (``react-page-frame01.png``)  or (``react-page-frame-01.png``)  than a space between name and digits because of editors wrapping its contents due to split screen.

**Comment by jokerfyy** [comment]
收到啦，请放心，我马上就会看的

**Comment by NickeManarin** [comment]
It will be available in v2.37.1
